### DISCUSSION TITLE
Request PostgreSQL Anonymizer extension

### DISCUSSION THREAD

**Discussion by magick93** [supabase/supabase#6789] [General]
Please add the Anonymizer extension. 

https://pgxn.org/dist/postgresql_anonymizer/0.12.0/

License - https://gitlab.com/dalibo/postgresql_anonymizer/-/blob/master/LICENSE.md

**Comment by steve-chavez** [comment]
Sounds interesting. I suggest you open an issue and ask for this one on https://github.com/supabase/postgres/
